File: Level 026/homework/hw2.py
# 4) შექმენით სია რომელშიც იქნება რენდომ რიცხვები მოთავსებული შემდეგ კი შექმენით მეორე სია რომელშიც გადაიტანთ პირველი სიიდან მხოლოდ ლუწ რიცხვებს

my_list = [12, 2309, 123, 2929, 20, 12345]
evens_list = []

for i in my_list:
    if i % 2 == 0:
        evens_list.append(i)

print(evens_list)

File: Level 071/homework/hw2.js
/* 3)შექმენით სია, ამ სიაში შეინახეთ String-ტიპის მონაცემები და for-ციკლის გამოყენებით ახალ სიაში დაამატეთ ძველი სიიდან ყოველი სიტყვის პირველი ასო */

let my_list = ["nika", "irakli", "giorgi", "guga", "askili"]
let new_list = []

for (i = 0; i < my_list.length; i++) {
    new_list.push(my_list[i][0])
}

console.log(new_list)